tasks/postinstall.yml

```diff
@@ -20,9 +20,18 @@
 - name: 'Install configuration logrotate files'
   template:
     src: openstack-keystone.conf.j2
-    dest: "/etc/logrotate.d/openstack-keystone"
+    dest: "{{ openio_keystone_logrotate_file }}"
     owner: root
     group: root
     mode: '0644'
   tags: configure
+
+- name: remove legacy wrong file
+  file:
+    path: /etc/logrotate.d/openstack-keystone
```

Comment:
why is this one hardcoded whereas the other is not ?

Reply:
Actually on bionic we provide a `/etc/logrotate.d/openstack-keystone`. This PR templatize the good file path and remove the legacy erroneous file path to avoid issue during migration

Reply:
Exactly what I was suspecting, you should put a comment there explaining just that...